.github/ISSUE_TEMPLATE/iceberg_docs_improvement

```diff
@@ -0,0 +1,69 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+---
+name: Iceberg Docs Site Improvement 📑
+description: Improvements to the Apache Iceberg Documentation, Specification, and/or Website
+labels: ["docs","site"]
+title: "[Docs]: "
+body:
+- type: markdown
+  attributes:
+    value: |
+      Thanks for contributing to Apache Iceberg docs ❤️ and taking time to fill out this form.
+      A quick primer on documentation in Iceberg:
+      * Documentation - refers to the versioned documentation that changes with each version of Apache Iceberg, maintained [in the `docs/` directory](https://github.com/apache/iceberg/tree/main/docs).
+      * Specification - refers to the core [Iceberg Specification](https://iceberg.apache.org/spec/), along with the [View](https://iceberg.apache.org/view-spec/), [Puffin](https://iceberg.apache.org/puffin-spec/), and [GCM stream specifications](https://iceberg.apache.org/gcm-stream-spec/) that are maintained [in the `format/` directory](https://github.com/apache/iceberg/tree/main/format).
```

Comment:
I'd regard specification as API, which requires formal proposal and discussion on the dev list. That said, it may also contain typo and invalid links. Maybe add a warning here?